components/chassis.py

```diff
@@ -270,15 +269,15 @@ def setup(self) -> None:
             self.kinematics,
             self.imu.getRotation2d(),
             self.get_module_positions(),
-            Pose2d(3, 0, 0),
+            Pose2d(0, 0, 0),
             stateStdDevs=(0.05, 0.05, 0.01),
             visionMeasurementStdDevs=(0.4, 0.4, math.inf),
         )
         self.field_obj = self.field.getObject("fused_pose")
         self.module_objs: list[wpilib.FieldObject2d] = []
         for idx, _module in enumerate(self.modules):
             self.module_objs.append(self.field.getObject("s_module_" + str(idx)))
-        self.set_pose(Pose2d(4, 3.5, Rotation2d.fromDegrees(180)))
+        self.set_pose(Pose2d(0, 0, 0))
```

Comment:
Oh, this would explain why we ended up inverting the drive motors :sweat_smile:

Could you flip `drive_reversed` back?

Reply:
Does this still need to be done @auscompgeek after we experimented , it seems like everything needs to be reversed or  that whole pipeline just isn't working for some reason

Reply:
I'm going to merge this as-is. I think we will need to come back and set the initial pose differently depending on what alliance we are on, so we'll need to do that later.